playbooks/prepare_stack.yaml

```diff
@@ -27,16 +30,31 @@
     environment:
       OS_CLOUD: standalone
 
-  - name: Create public network
+  - name: Create external network
     shell: |
-      if ! openstack network show public; then
-          openstack network create --external --share --provider-physical-network datacentre --provider-network-type flat public
+      if ! openstack network show external; then
+          openstack network create --external --share --provider-physical-network external --provider-network-type flat external
       fi
-      if ! openstack subnet show public-subnet; then
-          openstack subnet create public-subnet --subnet-range "{{ public_cidr }}" \
-              --no-dhcp --gateway "{{ public_gateway }}" \
-              --allocation-pool "start={{ public_fip_pool_start }},end={{ public_fip_pool_end }}" \
-              --network public
+      if ! openstack subnet show external-subnet; then
+          openstack subnet create external-subnet --subnet-range "{{ external_cidr }}" \
+              --no-dhcp --gateway "{{ external_gateway }}" \
```

Comment:
I think I enabled DHCP to make it work, but maybe in some networks it would work without?

Reply:
I'm not so sure about this. I guess we'll want to support provider networks both with and without dhcp. I'm going to leave DHCP disabled by default, but add a note to the README on how to enable it (it's a oneliner).

Reply:
Oh, wait. This is the external network. We definitely can't enable it there! @adduarte would have kittens 😂